quack_app/test/golden_widget_test.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter_test/flutter_test.dart';
```

Comment:
I get a warning when I run the tests locally: "Tag not specified". I tried doing some [background research](https://github.com/dart-lang/test/blob/master/pkgs/test/doc/configuration.md#configuring-tags), but not sure if it is necessary.